include/install-vitasdk.sh

```diff
@@ -9,9 +10,9 @@ install_vitasdk () {
 
   case "$(uname -s)" in
      Darwin*)
-      mkdir -p $INSTALLDIR
-      wget -O "vitasdk-nightly.tar.bz2" "$(get_download_link master osx)"
-      tar xf "vitasdk-nightly.tar.bz2" -C $INSTALLDIR --strip-components=1
+      mkdir -p $INSTALLDIR &&
+      wget -O "vitasdk-nightly.tar.bz2" "$(get_download_link master osx)" &&
+      tar xf "vitasdk-nightly.tar.bz2" -C $INSTALLDIR --strip-components=1 &&
```

Comment:
why add `&&`?

Reply:
Also it's possible to pipe `wget` into `tar` and avoid a temporary `.tar.bz` file on disk
```wget -O- ... | tar x -C ...```